### PR TITLE
[15.0][ADD] stock_barcodes_elaboration: New module for display elaborations in barcode interface

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa

### CODE REVIEW

File: stock_barcodes_elaboration/wizard/stock_barcodes_read_todo_view.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_stock_barcodes_todo_kanban" model="ir.ui.view">
+        <field name="model">wiz.stock.barcodes.read.todo</field>
+        <field name="inherit_id" ref="stock_barcodes.view_stock_barcodes_todo_kanban" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='state']" position="before">
+                <field name="elaboration_ids" optional="show" />
+                <field name="elaboration_note" optional="show" />
```
Comment (carlosdauden) [inline stock_barcodes_elaboration/wizard/stock_barcodes_read_todo_view.xml:9]: Kanban view

File: stock_barcodes_elaboration/wizard/stock_barcodes_read_picking_views.xml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            name="inherit_id"
+            ref="stock_barcodes.view_stock_barcodes_read_picking_form"
+        />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='pending_move_ids']/tree/button[@name='fill_from_pending_line']"
+                position="before"
+            >
+                <field name="elaboration_ids" widget="many2many_tags" />
+                <field name="elaboration_note" />
```
Comment (carlosdauden) [inline stock_barcodes_elaboration/wizard/stock_barcodes_read_picking_views.xml]:
```suggestion
                <field name="elaboration_ids" widget="many2many_tags" optional="show" />
                <field name="elaboration_note" optional="show" />
```

File: stock_barcodes_elaboration/wizard/stock_barcodes_read_todo_view.xml (hunk 1 above)
Comment (carlosdauden) [inline stock_barcodes_elaboration/wizard/stock_barcodes_read_todo_view.xml]: Kanban view.
```suggestion
                <field name="elaboration_ids" />
                <field name="elaboration_note" />
```

**Comment by elvise** [comment]
@carlosdauden can you rebase ?

**Comment by carlosdauden** [comment]
Rebased

**Comment by carlosdauden** [comment]
Ping @sergio-teruel @CarlosRoca13

File: stock_barcodes_elaboration/wizard/stock_barcodes_read_picking_views.xml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            >
+                <field name="elaboration_ids" widget="many2many_tags" optional="show" />
+                <field name="elaboration_note" optional="show" />
+            </xpath>
+            <xpath
+                expr="//field[@name='pending_move_ids']/kanban//field[@name='qty_done']"
+                position="after"
+            >
+                <field name="elaboration_ids" invisible="1" />
+<!--                <field name="elaboration_note" invisible="1"/>-->
```
Comment (CarlosRoca13) [inline stock_barcodes_elaboration/wizard/stock_barcodes_read_picking_views.xml]:
```suggestion
```

**Review by CarlosRoca13 — APPROVED** [review]
Code review :+1: 

Just a minor comment

File: stock_barcodes_elaboration/wizard/stock_barcodes_read_picking_views.xml (hunk 3 above)
Comment (carlosdauden) [reply stock_barcodes_elaboration/wizard/stock_barcodes_read_picking_views.xml]: Changes done

**Comment by carlosdauden** [comment]
ping @sergio-teruel ready to merge

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**